From: Kevin Roy <kiniou@gmail.com>
Date: Thu, 14 Apr 2011 13:26:34 +0200
Subject: [PATCH] install_in_usr_lib

 * Arch-specific files go to /usr/lib rather than /usr/share.
 * Make blender look for systemdir into /usr/lib instead of /usr/share
 * Remove blender_version from system directory result
---
 intern/ghost/intern/GHOST_SystemPathsX11.cpp |    2 +-
 source/blender/blenlib/BLI_path_util.h       |    2 +-
 source/blender/blenlib/intern/path_util.c    |    2 +-
 source/creator/CMakeLists.txt                |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/intern/ghost/intern/GHOST_SystemPathsX11.cpp b/intern/ghost/intern/GHOST_SystemPathsX11.cpp
index 71db005..28698b4 100644
--- a/intern/ghost/intern/GHOST_SystemPathsX11.cpp
+++ b/intern/ghost/intern/GHOST_SystemPathsX11.cpp
@@ -57,7 +57,7 @@ const GHOST_TUns8* GHOST_SystemPathsX11::getSystemDir() const
 {
 	/* no prefix assumes a portable build which only uses bundled scripts */
 #ifdef PREFIX
-	return (GHOST_TUns8*) PREFIX "/share";
+	return (GHOST_TUns8*) PREFIX "/lib";
 #else
 	return NULL;
 #endif
diff --git a/source/blender/blenlib/BLI_path_util.h b/source/blender/blenlib/BLI_path_util.h
index 1fd619d..0dc86cd 100644
--- a/source/blender/blenlib/BLI_path_util.h
+++ b/source/blender/blenlib/BLI_path_util.h
@@ -91,7 +91,7 @@ char *BLI_get_folder_version(const int id, const int ver, const int do_check);
 #define BLENDER_SYSTEM_FORMAT			"%s/Blender/%s"
 #else
 #define BLENDER_USER_FORMAT			"%s/.blender/%s"
-#define BLENDER_SYSTEM_FORMAT			"%s/blender/%s"
+#define BLENDER_SYSTEM_FORMAT			"%s/blender"
 #endif
 
 #ifdef WIN32
diff --git a/source/blender/blenlib/intern/path_util.c b/source/blender/blenlib/intern/path_util.c
index 4051ef0..a406031 100644
--- a/source/blender/blenlib/intern/path_util.c
+++ b/source/blender/blenlib/intern/path_util.c
@@ -992,7 +992,7 @@ static int get_path_system(char *targetpath, const char *folder_name, const char
 
 	system_base_path = (const char *)GHOST_getSystemDir();
 	if (system_base_path) {
-		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path, blender_version_decimal(ver));
+		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path);
 	}
 	
 	if(!system_path[0])
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 9c048a7..e6d79f7 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -338,7 +338,7 @@ if(UNIX AND NOT APPLE)
 		# install(CODE "message(\"copying blender scripts...\")")
 		install(
 			DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts
-			DESTINATION ${TARGETDIR_VER}/
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/blender/scripts
 			PATTERN ".svn" EXCLUDE
 			PATTERN "__pycache__" EXCLUDE
 		)
-- 
